programs/svm-spoke/src/lib.rs

```diff
@@ -2,10 +2,18 @@ use anchor_lang::prelude::*;
 
 declare_id!("Fdedr2RqfufUiE1sbVEfpSQ3NADJqxrvu1zojWpQJj4q");
 
-// External programs from idls directory (requires `anchor run generateExternalTypes`).
+// External programs from idls directory (requires anchor run generateExternalTypes).
 declare_program!(message_transmitter);
 declare_program!(token_messenger_minter);
 
+/// # Across SVM Spoke Prgram
```

Comment:
nit: shall it better go below just above the `#[program]` macro? not sure

Reply:
from the research I did it should be right below the `daclare_id` but before `#[program]` just like the `* @title` solidity block goes before the contract entry point.

Reply:
```suggestion
/// # Across SVM Spoke Program
```